Jenkinsfile

```diff
@@ -22,7 +22,7 @@ pipeline {
             steps {
                 dir('SyncIntegrationTests') {
                     sh 'pipenv install'
-                    sh 'pipenv check'
+                    sh 'pipenv check -i 36351'
```

Comment:
Could you add a comment mentioning https://github.com/pyupio/safety-db/issues/2272

Reply:
Ah sorry forgot you mentioned it, just added it